### PR TITLE
Fixes Issue #155 - Variable outside store

### LINKED ISSUE #155: [OZ] L-12 Variable outside store
**Please note that [OpenZeppelin](https://www.openzeppelin.com/) reported this issue.**

> In contrast to most of the code base, the [last policy identifier](https://github.com/neptune-mutual-blue/protocol/blob/73fc82fbe0d1388867b7df669983fe42760daeb1/contracts/core/policy/Policy.sol#L32) is saved directly in the `Policy` contract. However, to maintain continuity and prevent conflicts, any new version will [need to import](https://github.com/neptune-mutual-blue/protocol/blob/73fc82fbe0d1388867b7df669983fe42760daeb1/contracts/core/policy/Policy.sol#L35) the old value.
> 
> Consider saving it in the `Store` contract.

### PR DESCRIPTION
**Pull request by flashburst**
- Added key `NS_POLICY_LAST_PURCHASE_ID` to store the last policy id

### CODE REVIEW

File: contracts/libraries/PolicyHelperV1.sol [hunk 1]
```diff
@@ -274,4 +274,15 @@ library PolicyHelperV1 {
...
+
+  /**
+   * @dev Increases the "last policy id" and returns new id
+   *
+   */
+  function setLastPolicyId(IStore s) external returns (uint256 lastPolicyId) {
+    uint256 previous = s.getUintByKey(ProtoUtilV1.NS_POLICY_LAST_PURCHASE_ID);
+
+    lastPolicyId = previous + 1;
+    s.setUintByKey(ProtoUtilV1.NS_POLICY_LAST_PURCHASE_ID, lastPolicyId);
```
Comment (binodnpm) [inline contracts/libraries/PolicyHelperV1.sol:286]: Use `addUintByKey` instead of `get` and `set`.

File: contracts/libraries/PolicyHelperV1.sol [hunk 2]
```diff
@@ -274,4 +274,15 @@ library PolicyHelperV1 {
     // Fallback means the default option
     return COVER_LAG_FALLBACK_VALUE;
   }
+
+  /**
+   * @dev Increases the "last policy id" and returns new id
+   *
+   */
+  function setLastPolicyId(IStore s) external returns (uint256 lastPolicyId) {
```
Comment (binodnpm) [inline contracts/libraries/PolicyHelperV1.sol:282]: Rename to `incrementPolicyId`

Comment (binodnpm) [inline contracts/libraries/PolicyHelperV1.sol:282]: Rename to `incrementPolicyId`

**Comment by binodnpm** [comment]
@flashburst can you review this please?

**Review by rudolfnep — APPROVED** [review]

**Pull request merged**